package.json

```
@@ -4,7 +4,7 @@
   "description": "Balena PineJS client.",
   "main": "index.js",
   "types": "index.d.ts",
-  "browser": "es5",
+  "browser": "es2015/index.js",
```

I prefer
```suggestion
  "browser": "es2015",
```
unless there's some reason to specify the index.js?